src/main/java/edu/unifi/Main.java

```diff
@@ -40,11 +43,12 @@ public static void main(String[] args) throws InterruptedException {
         // Initiating database connection pool
         DatabaseAccess.initiate();
 
+
         loadingDialog.setVisible(false);
         try {
             Login loginView = new Login();
             loginView.getLoginLatch().await();
-            home = new Home("Test");
+            home = new Home("Test", loginView.getUsernameField().getText());
```

Comment:
not necessary, the username is contained in `CurrentSession.getInstance().getUser()`